lib/language_variants_filter.js

```diff
@@ -7,6 +7,8 @@ const mwUtil = require('./mwUtil');
 module.exports = (hyper, req, next, options, specInfo) => {
     const rp = req.params;
     const acceptLanguage = req.headers && req.headers['accept-language'];
+    // TODO: investigate why the following block is not allowing related to forward
+    // accept-language to summary request
     if (!acceptLanguage || mwUtil.isNoCacheRequest(req)) {
```

Comment:
/page/related endpoint is performing multiple requests to /page/summary, all of them pass through this filter. During debug, I could identify that, for some reason, if one of the requests hits isNoCacheRequest as true and delete accept-language header, all of the requests will be forwarded without the accept-language header.

If change line 12 with: `if (!acceptLanguage) {` the new added `related.js` test will pass (all tests pass).

Also, my patch assumes there is no other way to forward accept-language header since I'm not acquainted with restbase functionality.

Any thoughts @Pchelolo and @d00rman cc/ @mdholloway

Reply:
1. There's a bug here. We need to clone the request and deleted the headers from the clone. no-cache requests are used to rerender the stored content, and we only store default variant. So, we don't want to rerender a non-default variant. 
2. For your case of 'related' there shouldn't be no-cache requests - we don't store anything. Cache-control is removed by Varnish, so this header is under our internal control, thus there shouldn't be any no-cache requests for related.

Reply:
Done.